### ISSUE TITLE
怎么设置黑色背景

### ISSUE THREAD

**Issue by iCoobin92** [pkluz/PKHUD#228]
怎么设置黑色背景的风格

**Comment by PGLongo** [comment]
Really? Please write in english

**Comment by williamcaruso** [comment]
@iCoobin92 is asking how to change the style of the HUD to "dark mode." In reality this deserves to be its own feature.

Meanwhile, here is a hack way to do it, just change the following lines in the following files:

Window.swift | line 80
view.backgroundColor = UIColor(white:0.0, alpha:0.25) ~> view.backgroundColor = UIColor.black.withAlphaComponent(0.25)

FrameView.swift | line 26
backgroundColor = UIColor(white:1.0, alpha:0.36) ~> backgroundColor = UIColor.black.withAlphaComponent(0.36)

That should be it.

**Comment by PGLongo** [comment]
@williamcaruso I agree with you, but I'was answering with a little bit of irony to the message
